### PR TITLE
ignoring exceptions on stream close

### PR DESCRIPTION
**Pull request by eishay**
@raymondng

### CODE REVIEW

File: kifi-backend/modules/scraper/app/com/keepit/scraper/extractor/TikaBasedExtractor.scala [hunk 1]
```diff
@@ -58,7 +60,7 @@ abstract class TikaBasedExtractor(url: URI, maxContentChars: Int, htmlMapper: Op
         else
           log.error("extraction failed: ", e)
     } finally {
-      stream.close()
+      Try(stream.close())
```
Comment (raymondng) [inline kifi-backend/modules/scraper/app/com/keepit/scraper/extractor/TikaBasedExtractor.scala]: maybe log it so we can track via log entries?

**Comment by raymondng** [comment]
lgtm

**Comment by eishay** [comment]
Sure, will do

**Comment by eishay** [comment]
@raymondng done

**Comment by raymondng** [comment]
Thanks!

**Pull request merged**